sql/jjitTablesDeclare.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE jjit_job_offers (
+	id INT IDENTITY(1,1) PRIMARY KEY,
```

Comment:
Idea: Making the IDs UUIDs instead of ints

Reply:
Ints are much faster than guids when you compare them. We don't need the security of guids because the user won't have access to any benefits of knowing the id